eng/pipelines/templates/jobs/perf.yml

```diff
@@ -0,0 +1,116 @@
+parameters:
+  - name: serviceDirectory
+    type: string
+    default: ''
+  - name: services
+    type: string
+    default: ''
+  - name: packageVersions
+    type: string
+    default: '.*'
+  - name: tests
+    type: string
+    default: ''
+  - name: arguments
+    type: string
+    default: '' 
+  - name: iterations
+    type: number
+    default: '5'
+  - name: envVars
+    type: object
+    default: {}
+  - name: additionalArguments
+    type: string
+    default: ''
+
+variables:
+  - name: NugetSecurityAnalysisWarningLevel
+    value: 'none'
+  - name: skipComponentGovernanceDetection
```

Comment:
Similar to the comment on the NET PR, I think you can reference `globals.yml` instead.